examples/simple/priv/repo/migrations/20150710205000_create_city_and_country.exs

```diff
@@ -0,0 +1,43 @@
+defmodule Simple.Repo.Migrations.CreateCityAndCountry do
```

Comment:
No need to add this migration, just change the original one please. Nobody is running this in production. :D